pennylane/ops/op_math/parametrized_evolution.py

```diff
@@ -96,6 +96,12 @@ class ParametrizedEvolution(Operation):
         do_queue (bool): determines if the scalar product operator will be queued. Default is True.
         id (str or None): id for the scalar product operator. Default is None.
 
+    Keyword Args:
+        rtol (float, optional): relative local error tolerance for solver. Defaults to 1.4e-8.
+        atol (float, optional): absolute local error tolerance for solver. Defaults to 1.4e-8.
```

Comment:
I would add how the error is computed and what the absolute and relative tolerance mean. Something like the following where they are put together (saw this in the scipy docs):
"rtol, atol (float): Relative and absolute error tolerance. The error is estimated from comparing a 4th and 5th order Runge-Kutta step in the Dopri5 algorithm. This error is guaranteed to stay below ``tol = atol + rtol * abs(y)`` through adaptive step size selection."

Reply:
Added!

Reply:
Just a minor thing, it is not rendering as expected: 
![image](https://user-images.githubusercontent.com/43949391/214555529-e864c8ae-66c8-45d6-9339-247047657bd2.png)

Reply:
Fixed!